docs/traffic-plan.md

```diff
@@ -0,0 +1,9 @@
+# Traffic plan export API
+
+
+
+## Versions
+
+| Version                                  |
+|------------------------------------------|
+| [v3.0.0](openapi/trafficplan/index.html) |
```

Comment:
@eivindga var det snakk om at vi skulle liste opp versjoner på en annen måte? jeg husker ikke helt om det var noe vi pratet om. har bare kopiert oppsettet for MQTT i 2.x branchen, men det er kanskje ikke helt optimalt at versjonen er statisk her, men blir oppdatert av pipelinen i index.html.

Reply:
ja, sånn som det blir fremover for mqtt-api i adt, så linkes det her kun til en versjon. Så må man skifte i topp-menyen mellom releaser i stedet for at det står i tabellen. 

Se her feks:

https://github.com/RuterNo/adt-doc/blob/3.x/docs/adt-api.md?plain=1

Endret her fra tabell til en knapp som alltid peker på generert dokumentasjon, uten å nevne hvilken versjon.